### ISSUE TITLE
Specific sound does not play in the GameLauncher

### ISSUE THREAD

**Issue by LB-ArturZieba** [o3de/o3de#9050]
**Describe the bug**
In the project attached to this issue, the background music works fine in the Editor, but it does not play at all in the GameLauncher. This issue does not occur for any other sound, and reproduces with several different methods of including it in the level (different entity combinations, rewriting .scriptcanvas graphs etc.).

The background music setup can be found on the "Soundtrack" entity that is a child of the "Player" entity. The "Soundtrack" entity contains Audio Proxy, Audio Trigger (Send Finished Event enabled), and Script Canvas (MusicSC.scriptcanvas assigned) components.

Please note that the sound that shows the issue, as well as all the other sounds are custom-made sound files (.wav format) created in BoscaCeoil.

Reproduction requires a full project setup, but we could not find a workaround, nor a root cause of this issue.

**Steps to reproduce**
1. Download the project attached under the **Assets required** and build it in the Project Manager (Please note that the project was originally built on the Source engine configuration).
2. Launch the Editor.
3. Open the L1 level.
4. Enter the Game Mode.
5. Move around the level (W,A,S,D, and Space), note the music playing in the background, and the sounds of steps, jumping, as well as different pick ups.
6. Exit the Game Mode.
7. Launch the GJ413\build\windows\bin\profile\GJ413.GameLauncher.exe.
8. Choose the l1.spawnable from the .spawnable list.
9. Navigate around the level as in step 5.

**Expected behavior**
All the sounds work the same between the Editor and the GameLauncher.

**Actual behavior**
Background music does not play in the GameLauncher, even though the rest of the sounds work correctly.

**Assets required**
Project: [GJ413.zip](https://github.com/o3de/o3de/files/8521358/GJ413.zip)

**Video**

https://user-images.githubusercontent.com/86952082/164227755-b4ba5ffb-2779-4776-be00-3e2b7cb8f5d2.mp4

**Found in Branch**
Stabilization_2205 (95dbd13)

**Desktop**
 - Device: PC
 - OS: Windows
 - Version 10
 - CPU AMD Ryzen 5 3600
 - GPU NVIDIA GeForce RTX 2060 SUPER
 - Memory 16GB

**Comment by amzn-phist** [comment]
I think this is related to the `GameStateSamples` Gem.  I changed the volume in the pause > settings menu and all of a sudden heard the sound.  It's probably loading some default value and it's silent.  You wouldn't encounter this playing the level in Editor.

**Comment by amzn-phist** [comment]
@LB-ArturZieba 

Here's what I believe caused this issue:

When running the game launcher for the first time, there's no `GameOptions.savedata` stored on user's machine (if on Windows, this file should be saved to `%LOCALAPPDATA%\<ProjectName>.GameLauncher\`).  So the `GameStateSamples` Gem will use default values for "AmbientVolume", "EffectsVolume", "MainVolume", and "MusicVolume".  These all default to 100.0f in code.

The game loads and sends 100.0f for all four of these global parameters (RTPCs).  Opening up the Wwise project, only "MusicVolume" RTPC is defined out of these four and it's assigned to the `CMusic` sound.
![volume_rtpc_bug_0](https://user-images.githubusercontent.com/52085794/164789010-b7f12bbb-b0ac-43e6-8226-77635b100349.png)

If you look at the RTPC tab on that sound, you'll see a graph similar to this:
![volume_rtpc_bug_1](https://user-images.githubusercontent.com/52085794/164789071-3d4779ec-6210-458d-b82b-01dcba7da06a.png)

The X-axis is the "input" value, i.e. the value of the RTPC, and the Y-axis is the "output" value, i.e. the "Voice Volume" that will be set.  As you can see, setting a default value of 100.0f on it will attempt to produce a gain value of +200 dB!!  This is far too loud, and Wwise was probably seeing that and silencing the sound.  If you go into the game pause > settings menu and attempt to use the Music Volume slider, you can hear it get painfully loud as you drag the slider past the halfway point.  The way this RTPC is currently set up, the halfway point (0.5) on the slider would map to 0 dB, which is "full volume".

You will notice that after changing the slider to an acceptable volume, the `GameOptions.savedata` file will be saved, and subsequent plays of the game launcher will load that value and the music can be heard.

To fix this:

The RTPCs need to be fixed throughout this Wwise project.  Go through each of the sounds, look at the RTPC tab, and fix the ones that are mapped to "Voice Volume".

Click the top-right point on the graph (X: 100, Y: 200):
![volume_rtpc_bug_2](https://user-images.githubusercontent.com/52085794/164790261-3beabd51-e0ee-4346-a54e-e6d78e7b5231.png)

Enter a 0 value in the Y parameter and hit Enter:
![volume_rtpc_bug_3](https://user-images.githubusercontent.com/52085794/164790328-ff946c59-9b31-4e54-afab-31802e3c43c0.png)

Now save and generate banks.

**Comment by LB-ArturZieba** [comment]
Thanks for the explanation! I've confirmed on the Stabilization_2205 (968ce59) build that it was indeed the issue with RTPC value being set to maximum. 

However I've noticed that the `%LOCALAPPDATA%\` folder does not contain, nor generate a `<ProjectName>.GameLauncher\` folder of this GameLauncher. It should be noted that there was a `<ProjectName>.GameLauncher\` folder left after some project from a few months ago (deleting it did not cause the `GJ413.GameLauncher\` folder to be generated after launching the GJ413.GameLauncher), and the `%LOCALAPPDATA%\O3DE` did not contain any relevant files. Should this be reported as a separate issue, or could this .savedata file be generated to another location?

**Comment by amzn-phist** [comment]
In my experience, it did not create that folder (& file) automatically.  It wasn't until I entered the game's "Settings" menu and manually adjusted some of the sliders for audio volumes that the folder & file were created.  You may need to exit the menu as well, not exactly sure about that.  But if you verify that it does get created, you can also check that a subsequent run of the game will retain the slider values you set before (i.e. the settings get loaded properly).

**Comment by LB-ArturZieba** [comment]
I've reverified this as you described on the Stabilization_2205 (c8f1489) build, and the `<ProjectName>.GameLauncher\` is indeed generating along with the .savedata file after adjusting the audio sliders and navigating out of the Game Launcher Settings menu. The sliders are retained between launches, after the .savedata file is created. Deleting it, and rerunning the Game Launcher correctly regenerates it the same way as it did originally. 

I have also noticed that currently there is no documentation page describing the basics of the Game Launcher, so I have filed a relevant Docs Request here: https://github.com/o3de/o3de.org/issues/1588 if you are interested. Thanks again for the explanation!
